### PR TITLE
Adding vote to case

### PR DESCRIPTION
**Pull request by Mouroof**

### CODE REVIEW

File: cms/repository/postgres_objects.py [hunk 1]
```diff
@@ -59,6 +60,15 @@ class CaseComments(Base):
     commented__by = Column(String(40))
     comment_data = Column(JSON)
 
+class CaseVotes(Base):
+    __tablename__ = "t_case_votes"
+    vote_id = Column(String(40), primary_key=True)
+    case_id = Column(String(40), ForeignKey("t_case.case_id"))
+    voted__by = Column(String(40))
```
Comment (ik-nitk) [inline cms/repository/postgres_objects.py]: You did not change this to ForeignKey of member_id

File: application/rest/case.py [hunk 2]
```diff
@@ -73,6 +75,27 @@ def comments_list_api(id):
...
+    return Response(
+        json.dumps(response.value, cls=CaseJsonEncoder),
+        mimetype="application/json",
+        status=STATUS_CODES[response.type],
+    )
+
+@blueprint.route("/api/v1/cases/<id>/votes", methods=["GET"])
+def votes_list_api(id):
+    response = vote_list(current_app.config.get('REPO'), id)
+    print(response)
```
Comment (ik-nitk) [inline application/rest/case.py:93]: you can remove prints.

File: web/case/views.py [hunk 3]
```diff
@@ -59,10 +66,13 @@ def case_view(id):
         futures.append(pool.submit(get_case_details, api, session, id))
         futures.append(pool.submit(get_case_initial_documents, api, session, id))
         futures.append(pool.submit(get_case_comments, api, session, id))
+        futures.append(pool.submit(get_case_votes, api, session, id))
         wait(futures)
         case = futures[0].result()
         initial_doc_list=futures[1].result()
         case_comments=futures[2].result()
+        case_votes = futures[3].result()
+        print(case_votes)
```
Comment (ik-nitk) [inline web/case/views.py]: remove prints.

File: web/case/views.py [hunk 4]
```diff
@@ -126,6 +138,26 @@ def add_verification_details(case_id):
     except Exception as e:
         return render_template("error.html", error_msg=str(e))
 
+@blueprint.route("/cases/<case_id>/add_vote_to_case", methods = ["GET", "POST"])
+def add_vote_to_case(case_id):
+    try:
+        if request.method == "GET":
```
Comment (ik-nitk) [inline web/case/views.py:141]: as discussed in the call , remove this GET portion as its not needed.

File: cms/domain/case_votes.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+from cms.domain.vote_type import VoteType
+
+@dataclasses.dataclass
+class CaseVote:
+    vote_id:str
+    case_id: str
+    voted_by: str
+    amount_suggested:int
+    vote: VoteType
```
Comment (ik-nitk) [inline cms/domain/case_votes.py:11]: we would also want to have the comments here along with the vote type

**Review by ik-nitk — COMMENTED** [review]
can you please address the comments. and revert.

**Comment by Mouroof** [comment]
without GET portion how is it possible to go to case voting page.

File: web/case/views.py (hunk 4 above)
Comment (Mouroof) [reply web/case/views.py:141]: without GET portion how is it possible to go to case voting page.

Comment (ik-nitk) [reply web/case/views.py:141]: ok i see .. then np.  you can keep

Comment (Mouroof) [reply web/case/views.py:141]: ok

**Comment by Mouroof** [comment]
> can you please address the comments. and revert.

Addresed comments and reverted.

**Pull request merged**